app/grants/clr.py

```diff
@@ -350,10 +379,15 @@ def populate_data_for_clr(grants, contributions, phantom_funding_profiles, clr_r
         # phantom funding
         grant_phantom_funding_profiles = phantom_funding_profiles.filter(grant_id=grant.id, created_on__gte=clr_start_date, created_on__lte=clr_end_date)
 
-        # verified profiles
-        verified_profile_ids = [ele.pk for ele in contribs if ele.profile_for_clr.sms_verification]
-        verified_phantom_funding_profile_ids = [ele.profile_id for ele in grant_phantom_funding_profiles if ele.profile.sms_verification]
-        verified_profile = list(set(verified_profile_ids + verified_phantom_funding_profile_ids))
+        # SMS verified profiles
+        sms_verified_profile_ids = [ele.pk for ele in contribs if ele.profile_for_clr.sms_verification]
+        sms_verified_phantom_funding_profile_ids = [ele.profile_id for ele in grant_phantom_funding_profiles if ele.profile.sms_verification]
+        sms_verified_profile = list(set(sms_verified_profile_ids + sms_verified_phantom_funding_profile_ids))
+
+        # BrightID verified profiles
+        brightid_verified_profile_ids = [ele.pk for ele in contribs if ele.profile_for_clr.is_brightid_verified]
```

Comment:
@frankchen07 mapping this to https://github.com/gitcoinco/web/pull/7333/files#r485161156

Reply:
ossom, LGTM, makes sense.

- we have two lists
- clr calculations takes both
- we calculate multipliers based on set inclusions